commutilator_api/models.py

```diff
@@ -1,3 +1,44 @@
 from django.db import models
 
 # Create your models here.
+class TimeStamp(models.Model):
+    created_at = models.DateTimeField(db_index=True, auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        abstract = True
+
+
+class Vehicle(TimeStamp):
+    make = models.IntegerField(null=True)
+    model = models.IntegerField(null=True)
+    year = models.IntegerField(null=True)
+    mpg = models.IntegerField(null=True)
+
+
+class Commute(TimeStamp):
+    start_location = models.TextField(max_length=400)
```

Comment:
I thought TextFields didn't have a max and CharFields did. Will this throw an error?

Reply:
Just changed them to CharFields to be on the safe side.